CONTRIBUTING.md

```diff
@@ -11,39 +11,21 @@ This crate has some code paths that depend on Wasm Atomics, which has some prere
 - Cargo's [`build-std`].
 - The `atomics` and `bulk-memory` target features.
 
-Example usage:
-
-```sh
-# Installing Rust nightly and necessary components:
-rustup toolchain install nightly --target wasm32-unknown-unknown --component rust-src
-# Example `cargo build` usage:
-RUSTFLAGS=-Ctarget-feature=+atomics,+bulk-memory cargo +nightly build -Zbuild-std=panic_abort,std --target wasm32-unknown-unknown
-```
+These are set using [`rust-toolchain.toml`](./rust-toolchain.toml).
 
 ### Rust Analyzer
 
 To get proper diagnostics for Rust Atomics it can be helpful to configure Rust Analyzer to support
 that.
 
-Here is an example configuration for Visual Studio Code:
-
-```json
-"rust-analyzer.cargo.target": "wasm32-unknown-unknown",
-"rust-analyzer.cargo.extraArgs": [
-    "-Zbuild-std=panic_abort,std"
-],
-"rust-analyzer.cargo.extraEnv": {
-    "RUSTUP_TOOLCHAIN": "nightly",
-    "RUSTFLAGS": "-Ctarget-feature=+atomics,+bulk-memory"
-},
-```
+It takes the settings from `rust-toolchain.toml`, but we also need to specify a target, as seen for vscode in [.vscode/settings.json](./.vscode/settings.json).
```

Comment:
I don't want to assume that everybody uses VS Code, which is why I didn't include it in the repo.
My assumption is that it should be good enough to include this in the `CONTRIBUTING.md`, do you believe it is not?